factom_sdk/utils/utils.py

```diff
@@ -14,3 +14,21 @@ def generate_key_pair():
             A key pair object with Public and Private keys.
         """
         return KeyCommon.create_key_pair()
+
+    @staticmethod
+    def convert_raw_to_key_pair():

```

Comment:
@betoxd40 This might just be my Python ignorance, but do you need to specify the argument in this method, since we are actually passing in a required `raw_private_key` parameter, or is it ok to leave that blank since the method already allows for a blank value there? It does look a bit misleading to my eyes without specifying that parameter, but again, perhaps this is the convention in Python that I am unaware of (?).

Reply:
@dorono GOOD CATCH